src/test/scala/io/iohk/ethereum/ledger/BlockRewardSpec.scala

```diff
@@ -62,25 +93,47 @@ class BlockRewardSpec extends FlatSpec with Matchers with MockFactory {
     //- cake overrides
 
 
-    val validAccountAddress = Address(0xababab)
-    val validAccountAddress2 = Address(0xcdcdcd)
-    val validAccountAddress3 = Address(0xefefef)
+    val validAccountAddress = Address(0xababab)  // 11250603
+    val validAccountAddress2 = Address(0xcdcdcd) // 13487565
+    val validAccountAddress3 = Address(0xefefef) // 15724527
+
+    val validAccountAddress4 = Address("0x29a2241af62c0001") // 3000000000000000001
+    val validAccountAddress5 = Address("0x29a2241af64e2223") // 3000000000002236963
+    val validAccountAddress6 = Address("0x29a2241af6704445") // 3000000000004473925
 
     val minerTwoOmmersReward = BigInt("5312500000000000000")
     val ommerFiveBlocksDifferenceReward = BigInt("1875000000000000000")
+    val afterByzantiumNewBlockReward: BigInt = BigInt(10).pow(18) * 3
 
     val worldState: InMemoryWorldStateProxy = BlockchainImpl(storagesInstance.storages).getWorldStateProxy(-1, UInt256.Zero, None,
       noEmptyAccounts = false, ethCompatibleStorage = true)
       .saveAccount(validAccountAddress, Account(balance = 10))
       .saveAccount(validAccountAddress2, Account(balance = 20))
       .saveAccount(validAccountAddress3, Account(balance = 30))
+      .saveAccount(validAccountAddress4, Account(balance = 10))
+      .saveAccount(validAccountAddress5, Account(balance = 20))
+      .saveAccount(validAccountAddress6, Account(balance = 20))
 
     // We don't care for this tests if block is not valid
     def sampleBlock(minerAddress: Address, ommerMiners: Seq[Address] = Nil): Block = {
       Block(
         header = Fixtures.Blocks.Genesis.header.copy(beneficiary = minerAddress.bytes, number = 10),
         body = Fixtures.Blocks.Genesis.body.copy(
-          uncleNodesList = ommerMiners.map(a => Fixtures.Blocks.Genesis.header.copy(beneficiary = a.bytes, number = 5))
+          uncleNodesList = ommerMiners.map{ address =>
+            Fixtures.Blocks.Genesis.header.copy(beneficiary = address.bytes, number = 5)
+          }
+        )
+      )
+    }
+
+    def sampleBlockAfterByzantium(minerAddress: Address, ommerMiners: Seq[Address] = Nil): Block = {
+      val baseBlockNumber = BigInt(10).pow(18) * 3 + 1
```

Comment:
It is pretty big number for block number (now tests fail because of conversion `toInt` in reward calculator).
I think it would be the best to: 
Update `byzantium-block-number` in `mantis/src/test/resources/application.conf` to its real value (4375000)
And set this baseBlockNumber to`blockchainConfig.byzantiumBlockNumber`